node/test/go/dev-mode/test/integration/index.test.js

```diff
@@ -49,8 +48,20 @@ describe('Integration', function () {
       '4s-logger',
       {
         variants: new Map([
-          ['v16', { configuration: { Runtime: 'nodejs16.x', Timeout: 5 } }],
-          ['v18', { configuration: { Runtime: 'nodejs18.x', Timeout: 5 } }],
+          [
+            'v16',
+            {
+              ignoreMultipleInvocations: true,
```

Comment:
Why exactly this was added? It doesn't feel right

Reply:
@Danwakeem I had an issue with multiple invocations during tests in the past. The reason for that was the AWS account where I run the tests was integrated with Console and Console was changing the lambda config during the test, resulting in the creation of a new lambda container. If you run into a "multiple invocations" issue make sure you run the tests on an account not integrated with Console 👍

Reply:
🤔 Ahh okay maybe I was running it in the wrong account then. Let me try a different account and see if I no longer need this.

Reply:
Thanks @czubocha that was the issue 👍